pkg/gateway/api/api_grpc.go

```diff
@@ -1113,3 +1116,14 @@ func checkEnvironmentAPIKey(environmentAPIKey *accountproto.EnvironmentAPIKey, r
 func isContextCanceled(ctx context.Context) bool {
 	return ctx.Err() == context.Canceled
 }
+
+func (s *grpcGatewayService) filterOutArchivedFeatures(fs []*featureproto.Feature) []*featureproto.Feature {
+	result := make([]*featureproto.Feature, 0)
+	for _, f := range fs {
+		if f.Archived {
+			continue
+		}
+		result = append(result, f)
+	}
+	return result
+}
```

Comment:
This function was temporarily added. I will remove it when I implement a differential evaluation feature.